doc/configuration.rst

```diff
@@ -622,6 +623,67 @@ parameter::
 If the value of ``first_notebook_cell`` or ``last_notebook_cell`` is set to
 ``None``, then no extra first or last cell will be added to the notebook.
 
+.. _notebook_images:
+
+Adding images to notebooks
+==========================
+
+When notebooks are produced, by default (``notebook_images = False``) image
+paths from the `image` directive in rST documentation blocks (not images
+generated fom code) are included in markdown in their original form. This
+includes paths to images expected to be present on the local filesystem which
+is unlikely to be the case for those downloading the notebook.
+
+By setting ``notebook_images = True``, images will be included in the generated
+notebooks via Base64-encoded `data URIs <https://en.wikipedia.org/wiki/Data_URI_scheme>`_.
+As inclusion of images via data URIs can significantly increase size of the
+notebook, it's suggested this only be used when small images are used throughout
+galleries.
+
+An alternative is to instead provide a prefix string that'll be used for images
+e.g. the root URL of where your documentation is hosted. So for example the
+following configuration::
+
+    sphinx_gallery_conf = {
+        ...
+        'examples_dirs': ['../examples'],
+        'gallery_dirs': ['auto_examples'],
+        ...
+        'notebook_images': 'https://project.example.com/en/latest/',
+        ...
+    }
+
+with an example `image` directive in an rST documentation block being:
+
+.. code-block:: rst
+
+    .. image:: ../_static/example.jpg
```

Comment:
These docs should make it clear that it does a `relpath`-like computation relative to the docs root when building, so the `..` here takes us out of essentially `auto_examples` path where the SG-generated RST file lives. Without the `..`, the link would go to `https://project.example.com/en/latest/auto_examples/_static/example.jpg` (right?), which would be wrong.